### PR TITLE
profiles: last-rite most of sys-fabric/

### PR DESCRIPTION
**Pull request by SoapGentoo**
Signed-off-by: David Seifert <soap@gentoo.org>

### CODE REVIEW

File: profiles/package.mask [hunk 1]
```diff
@@ -33,6 +33,29 @@
 
 #--- END OF EXAMPLES ---
 
+# David Seifert <soap@gentoo.org> (2022-01-09)
+# No other distro packages these, long dead and unmaintained.
```
Comment (Flowdalic) [inline profiles/package.mask:37]: You may want to point out that at least some of those are now part of sys-cluster/rdma-core (sys-fabric/infiniband-diags for example).

Comment (SoapGentoo) [reply profiles/package.mask:37]: thanks, I will improve the message once the PR is green

File: profiles/package.mask [hunk 2]
```diff
@@ -33,6 +33,34 @@
 
 #--- END OF EXAMPLES ---
 
+# David Seifert <soap@gentoo.org> (2022-01-16)
+# Most of these packages have been merged into sys-cluster/rdma-core and
+# have long been dead and unmaintained.
+# Removal in 30 days, bug #601692.
```
Comment (jsmolic) [inline profiles/package.mask:39]: Could we also tag https://bugs.gentoo.org/830999 here

**Review by jsmolic — APPROVED** [review]
Thanks, let's ship this finally :tada:

**Comment by jsmolic** [comment]
Can you close https://bugs.gentoo.org/831003 with the 2nd commit while at it

**Pull request closed without merging**